tests/cli/test_state_diff.py

```diff
@@ -628,6 +628,326 @@ def test_diff_dhcp_addresses_are_filtered_out(self):
         missing = diff_data.get('interfaces', {}).get('eth0', {}).get('netplan_state', {}).get('missing_addresses', [])
         self.assertEqual(missing, [])
 
+    def test_diff_ra_addresses_are_filtered_out_with_accept_ra(self):
+        with open(self.path, "w") as f:
+            f.write('''network:
+  ethernets:
+    eth0:
+      dhcp4: true
+      accept-ra: true''')
+
+        netplan_state = NetplanConfigState(rootdir=self.workdir.name)
+        system_state = Mock(spec=SystemConfigState)
+
+        system_state.get_data.return_value = {
+            "netplan-global-state": {},
+            "eth0": {
+                "index": 4,
+                "id": "eth0",
+                "addresses": [
+                    {"10.86.126.253": {"prefix": 24, "flags": ["dynamic", "dhcp"]}},
+                    {"fd42:bc43:e20e:8cf7:8840:4dff:fea6:655d": {"prefix": 64, "flags": ["ra"]}},
+                    {"fe80::8840:4dff:fea6:655d": {"prefix": 64, "flags": ["link"]}}
+                ],
+                "dns_addresses": [
+                    "10.86.126.1",
+                    "fd42:bc43:e20e:8cf7::1",
+                    "fe80::216:3eff:feab:beb9"
+                ],
+                "routes": [
+                    {
+                        "to": "default",
+                        "family": 2,
+                        "via": "10.86.126.1",
+                        "from": "10.86.126.253",
+                        "metric": 100,
+                        "type": "unicast",
+                        "scope": "global",
+                        "protocol": "dhcp",
+                        "table": "main"
+                    },
+                    {
+                        "to": "10.86.126.0/24",
+                        "family": 2,
```

Comment:
nitpick: Consider using proper constants here and in the other tests below.

Reply:
hm I think I'll leave these as numbers. This is copied from state dumps and they will be represented as numbers. But I totally agree we should use the constants in non-test code.

Reply:
fair enough!